pyproject.toml

```diff
@@ -20,6 +20,8 @@ dependencies = [
     "marko==2.0.0",
     # Scripts
     "pandas==2.1.1",
+    # Gsheets
+    "pygsheets==2.0.6",
```

Comment:
I guess this should be added in #439 right?

Reply:
Thanks! Fixed